src/util/counter.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2022 The Yorkie Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ 'removeDecimalPoint' returns number with decimal removed
+ */
+export const removeDecimalPoint = (number: number) =>
```

Comment:
How about removeDecimal because it removes the decimal part from the number?

```suggestion
export const removeDecimal = (number: number) =>
```